games-action/nanosaur/nanosaur-1.4.3.recipe

```diff
@@ -0,0 +1,81 @@
+SUMMARY="Vintage 3D dinosaur shooter game"
+DESCRIPTION="This is Pangea Software’s Nanosaur updated to run on modern systems. Originally \
+released in 1998 as a Mac-only game, Nanosaur features a cybernetic dinosaur from the future \
+sent back in time 20 minutes before a giant asteroid hits the Earth."
+HOMEPAGE="https://pangeasoft.net/nano"
+COPYRIGHT="1998 Pangea Software, Inc."
+LICENSE="CC-BY-NC-SA-4.0"
+REVISION="1"
+
+srcGitRev="$portVersion"
```

Comment:
I think you can get rid of the srcGitRev stuff for the first source since it's downloading a tagged release?

Reply:
I added it for consistency because I saw something similar in another port. You're right of course, it isn't needed. I'll remove it